packages/tools/src/utilities/getAnnotationNearPoint.ts

```diff
@@ -99,8 +99,8 @@ function findAnnotationNearPointByTool(
   // the first annotation that is found in the proximity. BUT, we are not using
   // the function anywhere.
   const annotations = getAnnotations(
-    enabledElement.viewport.element,
-    (tool.constructor as typeof BaseTool).toolName
+    (tool.constructor as typeof BaseTool).toolName,
+    enabledElement.FrameOfReferenceUID
```

Comment:
We should have an exported function that provides the key to which state manager to use - that way one could have several different types of state manager for different purposes:
default - no FOR
volume - uses FOR and shows everywhere
volume - custom - uses FOR and display set key to key images to specific volumes
comparison - uses an alternate from default
If that is set on the element itself as a simple getter, then one passes in the key for what state manager to use, rather than assuming it will be FOR or nothing.